pandas/util/testing.py

```diff
@@ -299,18 +293,25 @@ def mplskip(cls):
     """Skip a TestCase instance if matplotlib isn't installed"""
 
     @classmethod
-    def setUpClass(cls):
+    def setup_class(cls):
         try:
             import matplotlib as mpl
             mpl.use("Agg", warn=False)
         except ImportError:
             import pytest
             pytest.skip("matplotlib not installed")
 
-    cls.setUpClass = setUpClass
+    cls.setup_class = setup_class
     return cls
 
 
+def _skip_module_if_no_mpl():
```

Comment:
This was for the matplotlib failures, correct? `tm.mplskip` is used on one class in `pandas/tests/io/formats/test_style.py`. From what I can see that's the only place it's used.

If you want, you could add a `pytest.importorskip('matplotlib')` on that test and delete `mplskip` altogether.